lib/content/audio/predictions.ex

```diff
@@ -161,7 +161,7 @@ defmodule Content.Audio.Predictions do
 
     defp four_cars?(audio) do
       PaEss.Utilities.prediction_four_cars?(audio.prediction) and !audio.terminal? and
-        !audio.multiple_messages?
+        !audio.multiple_messages? and audio.next_or_following == :next
```

Comment:
This would still generate a readout if the second prediction was read with "the next", which would happen if the destinations were different (e.g. Ashmont and Braintree). I think we'll need another flag or something, which we could set in `Message.Predictions.to_audio` based on the index.

Reply:
Ah thanks for catching that!